client/src/components/Home.svelte

```diff
@@ -39,45 +71,53 @@
     padding: 1rem;
     display: flex;
     justify-content: space-between;
-    align-items: center;
   }
 
-  .top-bar-left button,
   .top-bar-right button {
     background-color: #2a9d8f;
     color: white;
     padding: 0.5rem 1rem;
     cursor: pointer;
   }
 
-  .top-bar-left button:hover,
   .top-bar-right button:hover {
     background-color: #264653;
   }
 
+  .content-section{
+    height: 100vh;
+    width: 100vw;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    height: calc(100vh - 60px);
+  }
+
   .filter-section {
     background-color: #2a9d8f;
     padding: 2rem;
     color: white;
+		width: 25%;
```

Comment:
fix this excessive spacing, ill approve it for now but for your next pr please check for consistent spacing.